node_modules/nodeos-barebones/node_modules/nodeos-cross-toolchain/scripts/build

```diff
@@ -112,6 +112,8 @@ if [[ ! -d $OBJ_DIR ]]; then
         --with-sysroot=$TOOLS/$TARGET         \
         --disable-nls                         \
         --disable-shared                      \
+        --disable-target-libsanitizer         \
+        --disable-libsanitizer                \
```

Comment:
> it's a tool similar to valgrind

Thanks! :-D There's very little information about it besides was developed by Google...